### ISSUE TITLE
[LMS Admin] #4 NICE TO HAVE . wyszukiwarka plików

### ISSUE THREAD

**Issue by qunabu** [EscolaLMS/Admin#110]
NICE TO HAVE . wyszukiwarka plików

*Source url:* https://escolalms.github.io/Admin/#/Admin/files
*Reported by:* mateusz@escolasoft.com
*Reported at:* 13 Sep at 09:25 UTC

*Console:* [4× Error, 8× Log](https://ybug.io/dashboard/reports/detail/833j7jm1aaph63xc85ac#console)

*Location:* PL, Pomerania, Gdansk
*Browser:* Chrome 93.0.4577.63

*OS:* macOS 10.15.7
*Screen:* 1920x1080
*Viewport:* 1920x887

*Screenshot:*
![Screenshot](https://ybug.io/data/reports/b307ykjnw50dx3pg1r2y/833j7jm1aaph63xc85ac/screenshot.jpg?_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MzE1MjUxMzYsImRhdGEiOnsicmVwb3J0IjoiODMzajdqbTFhYXBoNjN4Yzg1YWMiLCJ1c2VyIjoiR2l0SHViIn19.yZO7_RC1z2pwelkYGe09kahXDbnkWX8l99yaFyeg3kk)

For more details please visit report page on Ybug:
[https://ybug.io/dashboard/reports/detail/833j7jm1aaph63xc85ac](https://ybug.io/dashboard/reports/detail/833j7jm1aaph63xc85ac)

**Comment by qunabu** [comment]
blocked by https://github.com/EscolaLMS/Files/issues/35

**Comment by qunabu** [comment]
there is new endpoint to find files/directores by search. 

- [ ] add search input next to upload new file 
- [ ] implement new endpoint https://escolalms.github.io/Files/#/Files/e44bf310b28fa6bb9db5ffdfde83bd0e
